fe/fe-core/src/main/java/com/starrocks/connector/delta/DeltaLakeMetadata.java

```diff
@@ -19,21 +19,21 @@
 import com.starrocks.catalog.HiveTable;
 import com.starrocks.catalog.Table;
 import com.starrocks.connector.ConnectorMetadata;
+import com.starrocks.connector.HdfsEnvironment;
 import com.starrocks.connector.hive.HiveMetastoreOperations;
-import org.apache.hadoop.conf.Configuration;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 
 import java.util.List;
 
 public class DeltaLakeMetadata implements ConnectorMetadata {
     private static final Logger LOG = LogManager.getLogger(DeltaLakeMetadata.class);
-    private final Configuration configuration;
     private final String catalogName;
     private final HiveMetastoreOperations hmsOps;
+    private final HdfsEnvironment hdfsEnvironment;
 
-    public DeltaLakeMetadata(Configuration configuration, String catalogName, HiveMetastoreOperations hmsOps) {
-        this.configuration = configuration;
+    public DeltaLakeMetadata(HdfsEnvironment hdfsEnvironment, String catalogName, HiveMetastoreOperations hmsOps) {
```

Comment:
seems it's not need to change the parameter from `Configuration` to `HdfsEnvironment`

Reply:
done